js/login.js

```diff
@@ -0,0 +1,13 @@
+import { checkEmail, checkPassword, updateSubmitButtonState, visibilityToggle } from "./authEventManager.js";
+import { email, password, inputs, visibilities } from "./authConstants.js";
+
+email.addEventListener('focusout', checkEmail);
+password.addEventListener('focusout', checkPassword);
+
+inputs.forEach((input) => {
+    input.addEventListener('focusout', updateSubmitButtonState);
+});
```

Comment:
💊 제안
동일한 input에 동일한 이벤트에 이벤트 바인딩을 진행하는데 이렇게 나눠서 작성해야하는 이유가 없을 것 같아요~

```suggestion
const checkInputValidation = (validator) => {
  validator();
  updateSubmitButtonState();
}

email.addEventListener('focusout', checkInputValidation(checkEmail));
password.addEventListener('focusout', checkInputValidation(checkPassword));
```

다만 이 구조는 validation 체크를 진행하고 무조건 input 값에 따라 버튼 상태를 업데이트하는 함수를 실행합니다~
실제로 유효성 검사를 한 후 실패한다면 버튼 상태를 체크할 필요가 없으니 이러한 것도 고민해보시면 좋겠습니다~

지금의 구조는 input의 유효성검사와 버튼의 상태를 업데이트하는 함수의 실행 시점이 다를 때 적절한 것 같습니다!